assembly/gradle-plugin/pom.xml

```diff
@@ -131,6 +131,7 @@
                                 <argument>-Pversion=${project.version}</argument>
                                 <argument>-Pdescription=${project.description}</argument>
                                 <argument>-S</argument>
+                                <argument>--no-daemon</argument>
```

Comment:
no-daemon for everyone? Does that impact the build time?

Reply:
@sschuberth could you answer that? I am unsure.

Reply:
It means that *this* run of Gradle will neither use nor spawn a Gradle daemon. It does not stop any running Gradle daemons. Esp. [for CI builds it's recommended to not use the Gradle daemon](https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:why_the_daemon) as the overhead of starting it does not pay off for one-time builds, and not using the daemon reduces memory usage.

Reply:
but this affects every build of antenna on every host, not only CI

Reply:
Ah, right. This was only a test anyway so see if it works around the [exit code 137 issue](https://stackoverflow.com/questions/38967991/why-are-my-gradle-builds-dying-with-exit-code-137) we saw on Jenkins. If not, we can remove it, if it does, then we should probably wrap it in a profile.